css/component_styles/card.css

```diff
@@ -0,0 +1,270 @@
+    background-color: var(--primary-hover);
+    color: #fff;
+}
+
+.card-badge-frame{
+    height: 53rem;
+}
+
+
+/* Card with Dismiss */
+    .button-close {
+      border: none;
+      width: 4%;
+      height: 8%;
+      position: absolute;
+      opacity: 0.7;
+      cursor: pointer;
+      z-index: 1;
+      outline: none;
+      right: 31%;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+}
+
+.card-dismiss-frame {
+    height: 55rem;
+}
+
+.overlay-text {
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 4rem;
+    font-weight: 300;
+    color: var(--primary-color);
+    opacity: 0.9;
+    margin-top: -30rem;
+}
+
+.overlay-container {
+    opacity: 0.3;
+} 
+
+.card-text-overlay-frame {
+    height: 65rem;
+}
+
+/* Text on Card */
+.center3 {
+    border-radius: 4px;
+    background-color: #fff;
+    box-shadow: 0 2px 1px -1px rgba(0, 0, 0, .2), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 1px 3px 0 rgba(0, 0, 0, .12);
+    display: flex;
+    width: 350px;
+    flex-direction: column;
+    box-sizing: border-box;
+    position: relative;
+    outline: none;
+    color: inherit;
+    text-decoration: none;
+    cursor: pointer;
+    overflow: hidden;
+    margin: auto;
+}
+
+.card {
+    padding: 1rem;
+}
+
+.head2 {
+    font-size: 1.9rem;
+    font-weight: 500;
+    letter-spacing: .0125em;
+    font-family: Roboto, sans-serif;
+    margin: 0;
+    line-height: 2rem;
+    text-decoration: inherit;
+    text-transform: inherit;
+}
+
+.head3 {
+    color: rgba(0, 0, 0, .54);
+    margin: 0;
+    font-size: 1.5rem;
+    line-height: 3rem;
+}
+
+.head3-2 {
+    color: var(--text-color);
+}
+
+.card2-text {
+    font-size: 1.2rem;
+    margin-left: 1rem;
+    color: var(--card-sub-text);
+}
+
+.footer {
+    width: 350px;
+    height: 52px;
+}
+
+.hover-content {
+    padding: 1rem;
+}
+
+.hover-content:hover {
+    background-color: var(--hover-aside);
+    border-radius: 0.7rem;
+
+}
+
+.hover-icon {
+    padding: 1rem;
+}
+
+.hover-icon:hover {
+    background-color: var(--hover-aside);
+    border-radius: 50%;
+}
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container3 {
+    display: flex;
+    justify-content: space-around;
+    font-size: 1.5rem;
+}
+
+.container3 a {
+    color: var(--card-text-color);
+}
+
+.card-text-frame{
+    height: 70rem;
+}
+
+/* Horizontal Card */
+.img_container {
+    display: flex;
+    flex-direction: row;
+    height: 110px;
+}
+
+.img1 {
+    width: 110px;
+    position: relative;
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-size: cover;
+}
+
+/* Vertical Card */
+.img_container2 {
+    position: relative;
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-size: cover;
+    color: inherit;
+    cursor: pointer;
+}
+
```

Comment:
more properties can be added to image base class for image (img1 & img_container2 has 4 same properties)